base/applications/network/telnet/src/tnerror.cpp

```diff
@@ -106,16 +163,21 @@ int printm(LPTSTR szModule, BOOL fSystem, DWORD dwMessageId, ...)
 				FORMAT_MESSAGE_FROM_STRING, szString, dwMessageId,
 				LANG_USER_DEFAULT, (LPTSTR)&pszMessage, 256, &Ellipsis);
 	}
+#endif
 
 	va_end(Ellipsis);
 
 	if (szModule)
 		FreeLibrary(hModule);
 
 	if (dwMessage) {
-
+#ifdef __REACTOS__
+		Result = wprintit(wszMessage);
+		LocalFree(wszMessage);
```

Comment:
.... there, only the wprintit would have to be inside the ifdef REACTOS, while the LocalFree could stay outside.